### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by daynewright**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>JS Quiz</title>
+    </head>
+    <body style="text-align: center; vertical-align: middle; margin-top: 15%;">
+        <h1>Input size of tree to create:</h1>
+        <form id="form" action="index.html" method="post">
```
Comment (gregkorte) [inline index.html]: Forms will automatically take care of an `Enter` press. For this exercise we want you to handle that on your own. I know you have a check for it, but the form is probably overriding it. The only way to be sure is to just get rid of the `<form>`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            <input type="text" name="height" id="height" value="" placeholder="amout of rows" onkeydown="checkKey(event)">
```
Comment (gregkorte) [inline index.html]: No running javascript from your html. You can achieve the same result by coding an event listener in your js file.

**Comment by gregkorte** [comment]
Nice job on your quiz, Dayne! Please see the above comments on your code. Nice logic in your createTree function, and I particularly like your error checking. Very thoughtful approach to this exercise. Let me know when you have your changes submitted, and I'll take another look.

**Comment by daynewright** [comment]
Thanks!  Added the changes.

**Comment by gregkorte** [comment]
You are good to merge, Dayne! Keep up the good work! :+1:

**Pull request merged**